recipes/relocate2/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: relocate2
+  version: 2.0.0
+
+source:
+  git_url: https://github.com/stajichlab/RelocaTE2.git
```

Comment:
Please use a tarball and a checksum.
This will always pull the latest github and if we rebuild the recipe it will not be the same.